packages-common/pluggable-widgets-generator/generators/app/templates/pluggable/web/fullTemplateTs/src/WidgetName.tsx.ejs

```diff
@@ -0,0 +1,33 @@
+import { Component, ReactNode, createElement } from "react";
+import { hot } from "react-hot-loader/root";
+
+import { <%= widgetName %>ContainerProps } from "../typings/<%= widgetName %>Props";
+import { BadgeSample } from "./components/BadgeSample";
+import "./ui/<%= widgetName %>.css";
+
+class <%= widgetName %> extends Component<<%= widgetName %>ContainerProps> {
+    private readonly onClickHandler = this.onClick.bind(this);
+
+    render(): ReactNode {
+        return (
+            <BadgeSample
+                type={this.props.<%= packageName %>Type}
+                bootstrapStyle={this.props.bootstrapStyle}
+                className={this.props.class}
+                clickable={!!this.props.onClickAction}
+                defaultValue={this.props.<%= packageName %>Value ? this.props.<%= packageName %>Value : ""}
+                onClickAction={this.onClickHandler}
+                style={this.props.style}
+                value={this.props.valueAttribute ? this.props.valueAttribute.displayValue : ""}>
```

Comment:
`this.props.valueAttribute?.displayValue ?? ""`

Reply:
Feedback reference included in a new story, so will be picked up in a follow up PR.